stdlib/LinearAlgebra/src/symmetric.jl

```diff
@@ -17,34 +17,45 @@ end
 Construct a `Symmetric` view of the upper (if `uplo = :U`) or lower (if `uplo = :L`)
 triangle of the matrix `A`.
 
+`Symmetric` views are mainly useful for real-symmetric matrices, for which
+specialized algorithms (e.g. for eigenproblems) are enabled for `Symmetric` types.
+More generally, see also `Hermitian(A)` for Hermitian matrices `A == A'`, which 
+is effectively equivalent to `Symmetric` for real matrices but is also useful for
+complex matrices.  (Whereas complex `Symmetric` matrices are supported but have few
+if any specialized algorithms.)
+
+To compute the symmetric part of a real matrix, or more generally the Hermitian part `(A + A') / 2` of
+a real or complex matrix `A`, use [`hermitianpart`](@ref).
```

Comment:
If we are going to keep the `symmetricpart` function (grr), then it would be weird not to mention it here as well.

Also, the `Hermitian` docstring should cross-reference `hermitianpart`.

Reply:
> If we are going to keep the `symmetricpart` function (grr), then it would be weird not to mention it here as well.

I don't think we are going to keep it. I just didn't want to be rude and simply remove it the moment I entered the discussion and "took over" the branch.

> Also, the `Hermitian` docstring should cross-reference `hermitianpart`.

I think that is in line 113.